test/TranslationLayer.UnitTests/TestSessionTests.cs

```diff
@@ -319,6 +320,23 @@ public void StopTestSessionShouldCallConsoleWrapperStopTestSessionWithCorrectArg
         _mockVsTestConsoleWrapper.Verify(
             vtcw => vtcw.StopTestSession(
                 _testSessionInfo,
+                null,
+                mockTestSessionEventsHandler2.Object),
+            Times.Once);
+    }
+
+    [TestMethod]
+    public void StopTestSessionShouldCallConsoleWrapperStopTestSessionWithCorrectArguments3()
```

Comment:
[minor] Could we try to find a different name than simply a number suffix? Past experience tells me that it's hard to keep the original intent with numbered "generic" names.

Reply:
Done, thanks !